release-notes/release-notes-2-4.md

```diff
@@ -1,29 +1,33 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 <!-- Copyright Contributors to the ODPi Egeria project. -->
 
-# Release 2.4 (Planned October 2020)
+# Release 2.4 (October 2020)
 
-Release 2.4 adds support for:
-   * Project management of IT change
-   * IoT and large scale cohorts
+* The [Integration Daemon](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/admin-services/docs/concepts/integration-daemon.md) now makes is simpler to exchange metadata with third party technology 
+  such as catalogs and databases.
+* The Repository explorer User Interface ('REX') can now be configured with a list of available 
```

Comment:
Need to be explicit that this is the presentation-server implementation of Rex.

Reply:
agreed - switched it around a bit (and can->must)